### PR TITLE
Freshchat Sdk upgrade to version 4.1.1

### PR DESCRIPTION
**Pull request by ravinder-groww**
I have upgraded the Freshchat react native SDK to version 4.1.1

### CODE REVIEW

**Review by sandeep-tiwari-groww — APPROVED** [review]

**Comment by ravinder-groww** [comment]
**We can merge** this PR now.
We should also merge relevant changes on Lucifer develop.

**Comment by abhaygroww** [comment]
@ravinder-groww to prevent a PR from getting merged, [mark it as a draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/), instead of commenting.

**Review by sagar-tomar-groww — APPROVED** [review]
Please verify that the reasons for forking the lib earlier are addressed.

File: android/gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,7 +1,6 @@
-#Wed Mar 18 16:06:58 IST 2020
-user=
+#Mon May 14 23:39:14 IST 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
```
Comment (anurag-dadheech) [inline android/gradle/wrapper/gradle-wrapper.properties:6]: Why?

File: android/gradle.properties [hunk 2]
```diff
@@ -1,3 +0,0 @@
-
-android.useAndroidX=true
-android.enableJetifier=true
```
Comment (anurag-dadheech) [inline android/gradle.properties:3]: Is this our change or done by library?

File: android/gradle/wrapper/gradle-wrapper.properties (hunk 1 above)
Comment (ravinder-groww) [reply android/gradle/wrapper/gradle-wrapper.properties:6]: I have done changes as per updated SDK which contains this distribution url only.

File: android/gradle.properties (hunk 2 above)
Comment (ravinder-groww) [reply android/gradle.properties:3]: This was our change which was done earlier. This is not present In library code.

**Review by anurag-dadheech — APPROVED** [review]

File: android/gradle.properties (hunk 2 above)
Comment (ravinder-groww) [reply android/gradle.properties:3]: Already discussed

**Comment by ravinder-groww** [comment]
> @ravinder-groww to prevent a PR from getting merged, [mark it as a draft](https://github.blog/2019-02-14-introducing-draft-pull-requests/), instead of commenting.

Addressed Here: https://github.com/Groww/lucifer/pull/3015#issue-1598260060

**Pull request merged**